module/zfs/zio.c

```diff
@@ -4260,15 +4260,15 @@ zio_dva_allocate(zio_t *zio)
 
 		if (zfs_flags & ZFS_DEBUG_METASLAB_ALLOC) {
 			zfs_dbgmsg("%s: metaslab allocation failure, "
-			    "trying normal class: zio %px, size %llu, error %d",
+			    "trying fallback: zio %px, size %llu, error %d",
```

Comment:
It would be nice if we could get the name of the fallback class somehow. A char buf in the metaslab_class_t with a human-readable name?

Reply:
> A char buf in the metaslab_class_t with a human-readable name?

Yea. Not a bad idea to think.